integration/volume_copy_up_test.go

```diff
@@ -70,7 +70,7 @@ func TestVolumeCopyUp(t *testing.T) {
 	assert.Equal(t, "test_content\n", string(stdout))
 
 	t.Logf("Check host path of the volume")
-	hostCmd := fmt.Sprintf("ls %s/containers/%s/volumes/*/test_file | xargs cat", *criRoot, cn)
+	hostCmd := fmt.Sprintf("ls %s/containers/%s/volumes/*/test_file | xargs cat", *criState, cn)
```

Comment:
Volume should be in persistent state.

How about we add "volatileStateRoot" and only change logic for `shm` and `fifos`, and keep other original logic.